Dockerfile

```
@@ -0,0 +1,85 @@
+    ca-certificates \
+    libssl3 \
+    && rm -rf /var/lib/apt/lists/*
+
+# Create non-root user
+RUN useradd -m -u 1000 -s /bin/bash synaptic
+
+# Create app directory
+WORKDIR /app
+
+# Copy built library from builder
+COPY --from=builder /usr/src/synaptic/target/release/libsynaptic.rlib /app/ || true
```

Hadolint (2.14.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)